src/main/java/hu/dpc/rt/kafkastreamer/exporter/KafkaExporterClient.java

```diff
@@ -74,9 +75,13 @@ public void index(final Record<?> record) {
 
         if (configuration.shouldIndexRecord(record)) {
             logger.trace("sending record to kafka: {}", record.toJson());
+            String jsonString = record.toJson();
+            String exportedTime = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format(new Date());
+            jsonString = jsonString.substring(0, jsonString.length() - 1)
+                         + ", \"exportedTime\": \"" + exportedTime + "\"}";
```

Comment:
Is this better?

Reply:
From the resource POV, Yes. Reduces one extra operation. And string manipulation should be a cheaper operation over parsing into a JSON Object, and then into the string.